.env

```diff
@@ -1,2 +1,3 @@
 REACT_APP_INSTANT_ENABLED=1
-REACT_APP_TAX_EXPORTER_ENABLED=1
\ No newline at end of file
+REACT_APP_TAX_EXPORTER_ENABLED=1
+REACT_APP_MKR_TOKEN_ENABLED=0
```

Comment:
Would be nice to apply those changes on `.env.dev` `.env.production`

Reply:
Actually I have the feeling we are not using those , can you verify please and if not, probably just delete the files :)

Reply:
I cannot verify because I have no idea how this app is deployed.
For my development adding to `.env` was enough

Reply:
Let 's leave it as it is then.